src/Pages/Register.cshtml

```diff
@@ -2,7 +2,7 @@
 @model ePiggyWeb.Pages.RegisterModel
 
 
-<div class="container login-container">
+<div class="container login-container col-lg-6 col-md-8 col-sm-10 col-12">
```

Comment:
I think 6 columns are a bit too large for desktop, I tried it out with 4 and it looks a bit more proportional in my opinion :)